cc3200/appsign.sh

```diff
@@ -13,7 +13,12 @@ BTYPE=$2
 BUILD=build/${BOARD}/${BTYPE}
 
 # Generate the MD5 hash
+# md5 on Darwin, md5sum on Unix
+if [ `uname -s` = "Darwin" ]; then
+echo -n `md5 $BUILD/application.bin | awk '{ print $4 }'` > __md5hash.bin
```

Comment:
Thanks for that. I was so focussed on simply making it run on my Mac I
didn't look any further.

Rgds

Gary

On Tue, Aug 25, 2015 at 6:58 AM, Antonin ENFRUN <notifications@github.com>
wrote:

> In cc3200/appsign.sh
> <https://github.com/micropython/micropython/pull/1438#discussion_r37803739>
> :
>
> > @@ -13,7 +13,12 @@ BTYPE=$2
> >  BUILD=build/${BOARD}/${BTYPE}
> >
> >  # Generate the MD5 hash
> > +# md5 on Darwin, md5sum on Unix
> > +if [ `uname -s` = "Darwin" ]; then
> > +echo -n `md5 $BUILD/application.bin | awk '{ print $4 }'` > __md5hash.bin
>
> You can use md5 -q $BUILD/application.bin to get only the checksum,
> without using awk
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/micropython/micropython/pull/1438/files#r37803739>.
>